projects/aca-playwright-shared/src/api/shared-links-api.ts

```diff
@@ -77,6 +77,19 @@ export class SharedLinksApi {
     }
   }
 
+  async shareNodes(nodeIds: string[], expiresAt?: Date): Promise<any> {
```

Comment:
can you already cleared function , shareFilesByIds

Reply:
removed it, and modified the existing function to be used in tests.